project/src/components/reviews-item/reviews-item.tsx

```diff
@@ -0,0 +1,38 @@
+import {CommentType} from '../../types/comments';
+
+type commentProps = {
+  comment: CommentType;
+}
+
+function ReviewsItem ({comment}: commentProps) : JSX.Element {
```

Comment:
memo

Reply:
если вся структура не нужна в компоненте, можно comment тоже деструктурировать прям тут типа 
```suggestion
function ReviewsItem ({comment: { user, rating, comment, date }}: commentProps) : JSX.Element {
```
чтоб не писать comment 7 раз